UPGRADE-7.0.md

```diff
@@ -5,6 +5,14 @@ Symfony 6.4 and Symfony 7.0 will be released simultaneously at the end of Novemb
 release process, both versions will have the same features, but Symfony 7.0 won't include any deprecated features.
 To upgrade, make sure to resolve all deprecation notices.
 
+Doctrine
```

Comment:
the title should be `Doctrine Bridge` IMO

Reply:
Yes!